src/wallet/test/wallet_tests.cpp

```diff
@@ -443,7 +444,8 @@ BOOST_FIXTURE_TEST_CASE(rescan, TestChain100Setup)
 // than or equal to key birthday.
 BOOST_FIXTURE_TEST_CASE(importwallet_rescan, TestChain100Setup)
 {
-    CWallet *pwalletMainBackup = ::pwalletMain;
+    LOCK(cs_main);
```

Comment:
This move of the cs_main lock is reverting parts of 95192d5b56737af0641f889878070e5c9cec3362

Reply:
done